proofs/cbmc/crypto_sign_keypair/Makefile

```diff
@@ -22,6 +22,7 @@ PROJECT_SOURCES += $(SRCDIR)/mldsa/sign.c
 CHECK_FUNCTION_CONTRACTS=$(MLD_NAMESPACE)keypair
 USE_FUNCTION_CONTRACTS=randombytes
 USE_FUNCTION_CONTRACTS+=$(MLD_NAMESPACE)crypto_sign_keypair_internal
+USE_FUNCTION_CONTRACTS+=$(MLD_NAMESPACE)mld_zeroize
```

Comment:
since this is a static function, you will have to remove the `$(MLD_NAMESPACE)`, I think.